S2VX.Game.Tests/UnitTests/S2VXStoryTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using NUnit.Framework;
+using osuTK;
+using S2VX.Game.Story;
+using S2VX.Game.Story.Note;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace S2VX.Game.Tests.UnitTests {
+    public static class S2VXStoryTests {
```

Comment:
See this section in our wiki for class vs function structure: https://github.com/maxrchung/S2VX/wiki/Testing#headlesstests. It's a bit out of date since at the time we didn't separate our tests as Unit, Headless, and Visual tests. "Headless" in the wiki's case refers to Unit tests.

Basically, any time there's an opportunity where you have common set up code and multiple asserts on that set up, I prefer the class structure. I think it better groups related test cases together and provides a common `[SetUp]` block for the test cases. Of course, the alternative is just using functions and helper functions, but I feel that having classes is nicer for encapsulating a set of assertions together.

Also as a note, we actually can't use test classes like this for our TestScene tests because osu!framework's TestScene detection is fairly primitive and can't detect nested test cases correctly.

`[TestFixture]` is not needed. NUnit is smart enough to detect a test based on the `[Test]` attribute.